bminiex : Working backlight (#4171)
qmk/qmk_firmware: keyboards/bminiex/readme.md

> B.mini

Reviewer: Should be `B.mini EX`.

========

> A 75% keyboard with RGB

Reviewer: Should specify something along the lines of this being a 75% + Numpad layout.

Author: I described it as "a compact fullsize keyboard with RGB" since it really has all keys to make a fullsize but the layout is different (clusters are squashed together).

Reviewer: That'll work.

> Keyboard Maintainer: QMK Community

Reviewer: This line needs to end in two spaces to be rendered as intended by GitHub.

```suggestion
Keyboard Maintainer: QMK Community  
```

> Hardware Supported: B.mini PCB

Reviewer: Should say `B.mini EX PCB`, also with two spaces at the end.

```suggestion
Hardware Supported: B.mini EX PCB  
```

> Hardware Supported: B.mini EX PCB

Reviewer: This line still needs two spaces at the end of it.

```suggestion
Hardware Supported: B.mini EX PCB  
```

> Hardware Availability: http://winkeyless.kr/product/b-mini-x2-pcb/

Reviewer: ```suggestion
Hardware Availability: https://winkeyless.kr/product/b-mini-ex-x2-pcb/
```

Make example for this keyboard (after setting up your build environment):

>     make bmini:default

Reviewer: ```suggestion
    make bminiex:default
```

> See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.

Reviewer: Please replace this line as follows:

```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
